feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/common/CloudBackupMetaAccountMapper.kt

```diff
@@ -0,0 +1,45 @@
+package io.novafoundation.nova.feature_account_impl.domain.common
+
+import io.novafoundation.nova.core_db.model.chain.account.ChainAccountLocal
+import io.novafoundation.nova.core_db.model.chain.account.MetaAccountLocal
+import io.novafoundation.nova.feature_cloud_backup_api.domain.model.CloudBackup
+import io.novafoundation.nova.feature_cloud_backup_api.domain.model.CloudBackupWallet
+
+fun mapLocalAccountToCloudBackup(modificationTime: Long, metaAccount: MetaAccountLocal, chainAccounts: List<ChainAccountLocal>): CloudBackup {
+    return CloudBackup(
+        modificationTime,
```

Comment:
Looks like this is outdated versino of CloudBackup model - latest one contains "Private" and "Public" data separately
Can you please merge base branch and fix conflicts?